lighthouse-core/test/config/config-helpers-test.js

```diff
@@ -18,12 +20,15 @@ const {
   mergePlugins,
   mergeConfigFragment,
   mergeConfigFragmentArrayByKey,
-} = require('../../config/config-helpers.js');
-const Runner = require('../../runner.js');
-const Gatherer = require('../../gather/gatherers/gatherer.js');
-const ImageElementsGatherer = require('../../gather/gatherers/image-elements.js');
-const UserTimingsAudit = require('../../audits/user-timings.js');
-const {LH_ROOT} = require('../../../root.js');
+} from '../../config/config-helpers.js';
+import Runner from '../../runner.js';
+import Gatherer from '../../gather/gatherers/gatherer.js';
+import ImageElementsGatherer from '../../gather/gatherers/image-elements.js';
+import UserTimingsAudit from '../../audits/user-timings.js';
+import {LH_ROOT} from '../../../root.js';
+import {createCommonjsRefs} from '../../scripts/esm-utils.js';
+
+const {require, __dirname} = createCommonjsRefs(import.meta);
```

Comment:
Gave it a shot. Would have to be done in `setup.js`, and would need a reference to the module being tested ... but that is not available.

Thought this might work but nah:

![image](https://user-images.githubusercontent.com/4071474/141001832-2f8a4d2c-0089-4b2d-aa61-0a1e9a5906ac.png)